src/content/docs/release-notes/agent-release-notes/java-release-notes/java-agent-7110.mdx

```diff
@@ -0,0 +1,119 @@
+---
+subject:  Java agent
+releaseDate:  '2022-10-27'
+version:  7.11.0
+downloadLink: 'https://download.newrelic.com/newrelic/java-agent/newrelic-agent/7.11.0/'
+---
+<ButtonGroup>
+  <ButtonLink
+    role="button"
+    to="https://download.newrelic.com/newrelic/java-agent/newrelic-agent/7.11.0/"
+    variant="primary"
+  >
+    Download this agent version
+  </ButtonLink>
+</ButtonGroup>
+
+## New features and improvements
+
+* Support Java 19 [1022](https://github.com/newrelic/newrelic-java-agent/pull/1022)
+* Support Play 2.8.16+ [981](https://github.com/newrelic/newrelic-java-agent/pull/981)
+* Support ojdbc8 v21.1.0.0+ [1042](https://github.com/newrelic/newrelic-java-agent/pull/1042)
+* Support Semeru/OpenJ9 JVMs [993](https://github.com/newrelic/newrelic-java-agent/pull/993)
+* Support log forwarding for java.util.logging (JUL) [1049](https://github.com/newrelic/newrelic-java-agent/pull/1049)
+
+* Support forwarding log context data [866](https://github.com/newrelic/newrelic-java-agent/pull/866)
+
+The agent can now forward data in Mapped Diagnostic Context (MDC, logback/slf4j) and ThreadContext (log4j2) as attributes when forwarding log records. When the feature is enabled, these attributes will be added to the log records with a `context.` prefix. For details on how to enable this feature see the documentation for [context_data](/docs/apm/agents/java-agent/configuration/java-agent-configuration-config-file/#logging-context-data).
```

Comment:
@meiao could you indent this four spaces since this material is about the preceding, indented bullet?